test/test_integration.py

```diff
@@ -472,7 +472,7 @@
             },
         ),
         (
-            "https://some.unknown/repo",
```

Comment:
Thanks for fixing this case. I hadn't realized it had been failing for that long.